test/remote/gateways/remote_pin_test.rb

```diff
@@ -17,14 +17,24 @@ def setup
       description: "Store Purchase #{DateTime.now.to_i}"
     }
 
-    @additional_options_3ds = @options.merge(
+    @additional_options_3ds_passthrough = @options.merge(
       three_d_secure: {
         version: '1.0.2',
         eci: '06',
         cavv: 'AgAAAAAAAIR8CQrXcIhbQAAAAAA',
         xid: 'MDAwMDAwMDAwMDAwMDAwMzIyNzY='
       }
     )
+
+    @additional_options_3ds = @options.merge(
+      three_d_secure: {
+        enabled: true,
+        fallback_ok: true,
+        callback_url: "https://yoursite.com/authentication_complete"
+      }
+    )
+  end
+
```

Comment:
Additional `end` here is causing a syntax error when trying to run remote tests, they pass with this removed.
```suggestion
```